sharpy/knowledges/knowledge_bot.py

```diff
@@ -65,7 +65,7 @@ def _create_start_msg(self) -> str:
 
         return msg
 
-    async def chat_send(self, message: str):
+    async def chat_send(self, message: str, team_only: bool = False):
```

Comment:
This is not used any where?

Reply:
Forgot to use it, fixed now. "Fixed" that before because PyCharm showed a warning there.